### ISSUE TITLE
On Windows, users must be instructed to patch the files in C:\Windows\System32\DriverStore\FileRepository\nvmdui.inf_amd64_ ...

### ISSUE THREAD

**Issue by doctorpangloss** [keylase/nvidia-patch#427]
Otherwise some applications will not load the patch.

**Comment by Snawoot** [comment]
I suppose it's true for drivers installed by Windows Updates, not by bundle from Nvidia site. Right?

**Comment by doctorpangloss** [comment]
I'm not 100% sure. I installed from NVIDIA's installer and I have these files. I also have the files in C:/Windows/System32.

Either way you have to patch them. I can see in Process Explorer that they're loaded by an application.

**Comment by doctorpangloss** [comment]
As a followup, anyone using the DCH drivers must update these. Most users will have DCH drivers.

**Comment by ajansveld** [comment]
@doctorpangloss I noticed the same, were you able to patch the files in this directory? Seems like it might be harder than just running the patcher as System.

**Comment by doctorpangloss** [comment]
No, I was not able to patch the DriverStore files. You cannot use this patcher with DCH drivers.

**Comment by doctorpangloss** [comment]
As an update, I have successfully patched these files by taking permissions of them.

**Comment by Kheopsian** [comment]
Precisely, how should we path these files, and what file ?
Because, I installed the new driver, patched it, did the bump for limit.
Using ffmpeg test i can start a lot of stream, but when I try plex transcoding, I'm still limited to 5

**Comment by doctorpangloss** [comment]
Install gsudo, then run the copy commands as Trusted Installer:
```
choco install -y gsudo
sudo --ti cp source.dll C:/Windows/.../destination.dll
```
